veps/sig-compute/vgpu_live-migration.md

```diff
@@ -0,0 +1,91 @@
+# VEP #109: Implement vGPU Enabled Live Migration
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [X] (R) Enhancement issue created, which links to VEP dir in [kubevirt/enhancements] (not the initial VEP PR)
+- [] (R) Target version is explicitly mentioned and approved
+- [] (R) Graduation criteria filled
+
+## Overview
+
+This is a proposal to allow live migrations in Kubevirt to work for VMs with a single NVIDIA vGPU. The live migration will be supported between two nodes with identical, Kubevirt compatible GPUs that use mdev.
+
+## Motivation
+
+GPU usage is increasing with more and more companies running AI workloads, so companies are now requesting live migration to support GPU enabled VMs.
+
+## Goals
+
+* Address a common live migration problem where the target needs to update the destination Libvirt XML. In this case, it needs to update the mdev UUID in the XML.
+* Support single vGPU enabled live migrations for both clusters that are using the Nvidia GPU Operator and clusters that are using Kubevirt’s generic device plugin for mdev.
+* Support single vGPU enabled live migrations with minimal data lost due to high dirty rates.
+
+## Non Goals
+
+Do not want to change the live migration workflow for non vGPU enabled VMs.
+
+## Definition of Users
+
+**Kubernetes Administrators:** Users who have cluster wide privileges to trigger APIs to manage a cluster.
+
+## User Stories
+
+As a cluster admin, I want to be able to live migrate my VMs that have a vGPU.
+
+## Repos
+
+https://github.com/kubevirt/kubevirt
+
+## Design
+
+In order to live migrate a vGPU, the destination XML needs to be updated with the mdev UUID that Kubevirt allocates on the target virt-launcher. To do this, we introduce a QEMU hook that interacts with the target `virt-launcher` pod.
+
+This will be the flow:
```

Comment:
I wonder if you could describe a bit more about how this mechanism will work. 
If I understand correctly, the idea is to rely https://libvirt.org/hooks.html#qemu-guest-migration

Is the idea to completely bypass the existing [migratableDomXML](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-launcher/virtwrap/live-migration-source.go#L198) ?
If not , then how will the approaches not collide?